### PR TITLE
APPDEV-1212 Swift 4.2 migration

### PR DESCRIPTION
**Pull request by vishal-revdiwala**
- Updated Swift 4.2 syntax

### CODE REVIEW

**Comment by vishal-revdiwala** [comment]
Before merging this PR, we need to upgrade build environment environment in mac-os agent. If you are ok with changes, I'll go ahead do the required updates.

File: Yona/Yona/Others/AVPageViewController.swift [hunk 1]
```diff
@@ -12,9 +12,15 @@ class AVPageViewController: UIPageViewController, UIPageViewControllerDataSource
     
     var contentViews: Array<UIViewController>?
     var presentingIndex: Int = 0
+//
```
Comment (Bert-R) [inline Yona/Yona/Others/AVPageViewController.swift]: Please remove this block of dead code

**Comment by vishal-revdiwala** [comment]
IDE highlight error for new Swift version. Manually have to fix error.

Also recent build from develop got failed, due to old Xcode version installed in mac-os agent. Once I update to latest Xcode 10.1, it will start to work. Shall I go ahead and do update in mac-os agent?

**Comment by Bert-R** [comment]
So fixing these incompatibilities is indeed a cumbersome task. Shame on Apple!

Yes, please upgrade the build agent and describe the change on the wiki page that documents the machine configuration.

**Comment by vishal-revdiwala** [comment]
Yeah sure.

**Comment by Bert-R** [comment]
Please comment here when the build agent updates are done.

**Comment by vishal-revdiwala** [comment]
Build agent is updated.

**Pull request merged**